internal/http/check_until_broker_ready_test.go

```diff
@@ -29,18 +29,18 @@ func TestCheckUntilBrokerReadyHappyPath(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			requestCount := 0
-			server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Personally I'm not a big fan of abbreviating names too much. There's plenty of room in the world for characters to have clear and explicit names :D

Reply:
Agree! Following this as I think it's the Go style.